### ISSUE TITLE
epoch setting for the CC-Finetune text supervised contrastive training stage

### ISSUE THREAD

**Issue by Gbone3176** [microsoft/LLM2CLIP#63]
Hello, this is truly a wonderful and inspiring work. During the reproduction process, I am a bit confused about the epoch setting of 1 for the CC-Finetune pure text fine-tuning stage. Why is **only 1 epoch** sufficient to learn enough text (mainly similar subtitle text here) representation ability? [related configs here](https://github.com/microsoft/LLM2CLIP/blob/cba3b45ba3d9bdaaf8ad640156dedbdbcb60353a/llm_caption_contrastive/train_configs/supervised/MetaLlama3_cc3m.json#L13]) 

Looking forward to your reply. Thanks again.

**Comment by raytrun** [comment]
Thanks! In our experiments, 1 epoch is already enough for convergence in the pure text finetuning stage. We didn’t see clear benefits from more epochs, so we chose to allocate the compute to more data instead.

**Comment by Gbone3176** [comment]
Thank you for your response. Are you referring to the benefits you mentioned as the direct results on the downstream CLIP task, or are you referring to the loss or other metrics during the Stage 1 training phase? I don't know much about representation learning, so I'm not very clear on these things. Thank you very much for your reply!

**Comment by raytrun** [comment]
Thanks for the question. The benefits I mentioned refer to the improvements on the downstream CLIP tasks,
